reactjs-films-homework/src/components/Header/Header.stories.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
-
 import { Story } from '@storybook/react';
+
 import Header from './Header';
 
+import { Provider } from 'react-redux';
+import store from '../../store/store';
+
+const Template: Story<typeof Header> = () => (
+  <Provider store={store}>
+    <Header />
+  </Provider>
+);
+
+export const HeaderStory = Template.bind({});
```

Comment:
you can do it shorter way:
`
export const header = (): JSX.Element => (
  \<Provider store={store}\>
    \<Header /\>
  \</Provider\>
);

export default {
  title: 'Header',
};
`